keyboards/gray_studio/hb85/program

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
```

Comment:
RE #5259, your rules.mk call out to `./keyboards/ps2avrGB/program $(TARGET).hex` so this file is a duplicate and not required.

Reply:
@zvecr Thanks a lot for your review! I will be starting all the proposed changes now and will reply as soon as they are ready.